tests/source/issue-3343.rs

```diff
@@ -15,6 +15,31 @@ extern crate len_is_50_;
 #[cfg(feature = "alloc")]
 extern crate len_is_51__;
 
+// https://github.com/rust-lang/rustfmt/issues/3343#issuecomment-589945611
+extern "C" {
+    #[no_mangle]
+    fn foo();
+}
```

Comment:
Let's expand on these test cases:
1. I'd like to see tests that include comments between the attribute and rewritten AST node.
2. I'd like to see tests cases with more than 1 attribute.